packages/core/src/compose/client.ts

```diff
@@ -36,8 +38,13 @@ const composeClient = (
   const getModel = async () => yaml.parse(await execComposeCommand(['convert'])) as ComposeModel
 
   return {
-    startService: (...services: string[]) => execComposeCommand(['start', ...services]),
     getModel,
+    getModelOrUndefined: async () => getModel().catch(e => {
```

Comment:
Is that a fix for the missing compose file?

Reply:
Also, currently I don't see any usage of that.

Reply:
It gets used by the incoming plugins code.